django/db/models/sql/query.py

```diff
@@ -1418,7 +1460,7 @@ def setup_joins(self, names, opts, alias, can_reuse=None, allow_many=True):
             reuse = can_reuse if join.m2m else None
             alias = self.join(connection, reuse=reuse)
             joins.append(alias)
-        return final_field, targets, opts, joins, path
+        return final_field, targets, opts, joins, path, final_transformer
```

Comment:
No argument here, I got a bit of underscore blindness at times. Think that should be its own PR, so we don't have too much refactoring in with functionality changes?